starting_from_scratch_lab/lab_40/tar2.js

```diff
@@ -0,0 +1,5 @@
+const selectElement = document.querySelector(".source");
+selectElement.addEventListener("change", (event) => {
+  var inputs = document.querySelectorAll("input");
+  inputs.forEach((input) => (input.value = `${event.target.value}`));
```

Comment:
1. No need to wrap `event.target.value` with an expression literal, as the value is either a string or a number, which can be simply assigned to the input.

2. In arrow functions there are multiple ways to wrap the function body:
- `(` round brackets or nothing = return that value (this is what you are doing)
- `{` normal function brackets behave like a normal function execution.

For example:
`(item) => ({ firstItem: item })` is returning an object and the same as
`(item) => { return { firstItem: item }}`
`(item) => { firstItem: item }` will give you an error.
`(item) => item` will simply return item.

Reply:
This is how I would write it eventually:

```js
const selectElement = document.querySelector(".source");
const inputs = document.querySelectorAll("input");

selectElement.addEventListener("keyup", (event) => {
  inputs.forEach((input) => {input.value = event.target.value});
});
```